public/app/plugins/datasource/loki/querybuilder/LokiQueryModeller.ts

```diff
@@ -25,13 +25,31 @@ export class LokiQueryModeller extends LokiAndPromQueryModellerBase<LokiVisualQu
     return super.renderLabels(labels);
   }
 
-  renderQuery(query: LokiVisualQuery) {
+  renderQuery(query: LokiVisualQuery, nested?: boolean) {
```

Comment:
As both `renderQuery` and `hasBinaryOp` are the same for both prom/loki maybe we could just leave it in the abstract base class.

Reply:
done, was a bit tricky as the types where different